LoopKitUI/ViewModels/TherapySettingsViewModel.swift

```diff
@@ -11,37 +11,37 @@ import LoopKit
 import HealthKit
 import SwiftUI
 
-public typealias SyncSchedule = (_ items: [RepeatingScheduleValue<Double>], _ completion: @escaping (Swift.Result<BasalRateSchedule, Error>) -> Void) -> Void
-public typealias SyncDeliveryLimits = (_ deliveryLimits: DeliveryLimits, _ completion: @escaping (_ result: Swift.Result<DeliveryLimits, Error>) -> Void) -> Void
+public enum SaveTherapySettingsError: Error {
+    case cancelTempBasalError(Error)
+    case missingPumpManager
```

Comment:
Maybe this enum goes away, if we don't need the missingPumpManager case (see other PR).

Reply:
Sure, but we still need *some* error meaning "CancelTempBasal failed", wrapping around the reason for it to fail.